app/src/org/commcare/android/database/user/UserDatabaseUpgrader.java

```diff
@@ -214,7 +217,33 @@ private boolean upgradeSevenEight(SQLiteDatabase db, int oldVersion, int newVers
      * attributes.
      */
     private boolean upgradeEightNine(SQLiteDatabase db, int oldVersion, int newVersion) {
-        return FixtureSerializationMigration.migrateFixtureDbBytes(db, c);
+        Log.d(TAG, "starting user fixture migration");
```

Comment:
Is this identical to the other method other than one or two parameters? Feels like there's a lot of repeated code.